database/migrations/2022_02_20_075758_create_jobs_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateJobsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('jobs', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('queue')->index();
+            $table->longText('payload');
+            $table->unsignedTinyInteger('attempts');
+            $table->unsignedInteger('reserved_at')->nullable();
+            $table->unsignedInteger('available_at');
+            $table->unsignedInteger('created_at');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('jobs');
+    }
+}
```

Comment:
i believe we only need this migration if the jobs are being tracked via the database, but we're using Redis to en-queue jobs, no?

Reply:
I had done it with database and assumed you would elaborate on replacing it with Redis in your initial review after we spoke about it briefly. My assumption was that since you didn't it was fine as-is but I can make the switch to Redis if it's better.

Reply:
i see, yeah, I suppose it doesn't matter, this just means that if a developer doesn't have redis running locally they can fall back to the database driver for enqueing jobs, let's leave it